Assets/Tests/InputSystem/CoreTests_GlobalActions.cs

```diff
@@ -0,0 +1,500 @@
+#if UNITY_INPUT_SYSTEM_ENABLE_GLOBAL_ACTIONS_API
+
+using System;
+using System.IO;
+using System.Linq;
+using System.Text.RegularExpressions;
+using NUnit.Framework;
+using UnityEditor;
+using UnityEngine;
+using UnityEngine.InputSystem;
+using UnityEngine.InputSystem.Editor;
+using UnityEngine.InputSystem.Interactions;
+using UnityEngine.InputSystem.Utilities;
+using UnityEngine.TestTools;
+using Input = UnityEngine.InputSystem.Input;
+
+internal partial class CoreTests
+{
+    private const string TestCategory = "GlobalActions";
+    private string m_TemplateAssetPath;
+
+    [SetUp]
+    public override void Setup()
+    {
+        // this asset takes the place of InputManager.asset for the sake of testing, as we don't really want to go changing
+        // that asset in every test.
+        var testInputManager = ScriptableObject.CreateInstance<TestInputManager>();
+        AssetDatabase.CreateAsset(testInputManager, "Assets/TestInputManager.asset");
+
+        // create a template input action asset from which the input action asset stuffed inside the InputManager will be created
+        var globalActions = ScriptableObject.CreateInstance<InputActionAsset>();
+        globalActions.AddActionMap("ActionMapOne").AddAction("ActionOne");
+        m_TemplateAssetPath = Path.Combine(Environment.CurrentDirectory, "Assets/TestGlobalActions.inputactions");
+        File.WriteAllText(m_TemplateAssetPath, globalActions.ToJson());
+
+        InputSystem.SetGlobalActionAssetPaths(m_TemplateAssetPath, "Assets/TestInputManager.asset");
```

Comment:
This seems to be overriding what the global asset points at.
I'm wondering why we can't just use/test the global asset default bindings

Reply:
I see test below forcing in additional bindings and then triggering specific control activations.
We could potentially change these to expected default binding control activations and remove the custom override.

However this isn't critical for 0.8.0-pre and might require platform specific binding checks